docs/WebSocket Transport/index.md

```diff
@@ -69,25 +73,32 @@ SDL Provides a JSON Response
 
 
 #### WebSocket Connection Diagram
+|||
```

Comment:
Looks like there should be a line break before the diagram.

Reply:
The line breaks aren't really consistent between documentation files(for example UI/ScrollableMessage has no extra line breaks between the diagrams). To make it consistent for this file I'm removing the extra line break for the `HMI Adapter Initialization` sequence diagram